flight/drone/drone_controller.py

```diff
@@ -243,3 +267,22 @@ def _land(self):
             sleep(c.DELAY_INTERVAL)
         self._logger.info('Disarm complete')
         self._logger.info('Finished land')
+
+    def _gather_data(self):
+        """Puts all gatherable data into a dictionary.
+
+        Notes
+        -----
+        Used to send a dictionary of data to a DataSplitter object, which logs
+        and potential graphs the data.
+
+        Returns
+        -------
+        dict
+        """
+        data = {}
+        for attribute, function_suffix in c.ATTRIBUTE_TO_FUNCTION.iteritems():
+            data[attribute] = eval("self._drone.{}".format(function_suffix))
```

Comment:
I don't think this is gonna fly. Is there no other way to solve this problem without using `eval`?

Reply:
I can't think of another way to do it that won't require the constants folder somehow having a reference to the drone. Alternatively, the dictionary ATTRIBUTE_TO_FUNCTION could be a constant member variable of DroneController, which would avoid the eval situation, but it would add some messiness to DroneController.py. Thoughts?